README.md

```diff
@@ -315,6 +315,22 @@ Test.ps1 | This is the PowerShell script to test that the uploaded ISO matches t
 Feel free to modify these files to suit your specific needs.
 General changes that you want to contribute back via a pull request are much appreciated.
 
+## Default GreenlakeAdmin password
```

Comment:
"Lake" is capitalized in the user data 
```suggestion
## Default GreenLakeAdmin password
```

Reply:
That was actually a typo. In the cloudbase-init config when we tell it what account to rename to it is "GreenlakeAdmin" so that's what the user is renamed to. I have updated the value in the user-data file to reflect what it really is and left the Readme alone.

Reply:
I assume it is not case sensitive? I was using GreenLakeAdmin and it worked (at least the last time when I got it to meet the complexity criteria).